doc-botonic/docs/faq.md

```diff
@@ -0,0 +1,36 @@
+---
+id: faq
+title: FAQ
+---
+
+
+><details><summary><b>How does Botonic and Hubtype relate to each other?</b></summary>
+Botonic is an open source project created by the Hubtype team. Hubtype is a startup that helps companies engage with their customers on messaging apps.
+Hubtype offers the best developer experience when it comes to deploying and integrating your Botonic bot.
+</details>
+
+><details><summary><b>How do I integrate a chat in a website?</b></summary>
+Once you deploy a bot,  you get a link to integrate different channels, including the "webchat" channel, which is a JavaScript snippet that you can embed in your website.
+</details>
+
+><details><summary><b>Can I edit my chat  styles?</b></summary>
+This action is not possible except for webviews as some of these apps (like Facebook Messenger) allow you to embed html pages inside their chat window.
+Hubtype provides support for web chat (JS SDK) and in-app chat (iOS/Android SDK). In these cases, you can customize the look and feel of the chat window.
```

Comment:
I would rephrase this by saying that "yes, you can ..." and at the end adding the exception "however, you can't change the style of messaging apps like Whatsapp... in these cases you can only customise webviews..."

Reply:
- The menu problem is resolved.
- Detailed API component not added in this PR. I will add it for the next sprint.
- doc-botonic/docs/faq.md updated.